_data/contributors.yml

```diff
@@ -356,3 +356,13 @@
   packages-submitted: ["physcraper"]
   packages-reviewed: [""]
   packages-editor: [""]
+ - name: Alejandro Sáez Mollejo
```

Comment:
@AlexS12 do you mind removing this extra whitespace before "-" ?
Other, LGTM! thanks!

Reply:
Sure! Should be ok now